packages/variable-editor/src/components/variables/dialog/AddDialog.tsx

```diff
@@ -106,18 +130,24 @@ export const AddVariableDialog = ({ table }: AddVariableDialogProps) => {
           </BasicField>
         </Flex>
         <DialogFooter>
-          <DialogClose asChild>
-            <Button
-              variant='primary'
-              size='large'
-              type='submit'
-              aria-label='Create variable'
-              disabled={!allInputsValid()}
-              onClick={addVariable}
-            >
-              Create Variable
-            </Button>
-          </DialogClose>
+          <Flex>
+            {knownVariable && (
+              <Message variant='info' message='This Variable is already present in a required project. Do you want to import it?' />
+            )}
```

Comment:
I don't like this message left, button right style.
Why not add this message as last child of the main dialog content below the last field.
And I think this can be a warning, then it is more present.

Reply:
![image](https://github.com/user-attachments/assets/c9614bf7-92cf-4bb2-81a3-48567f3e54b5)